### ISSUE TITLE
2.0: Backwards Compatibility Testing

### ISSUE THREAD

**Issue by JJJ** [sugarcalendar/sugar-event-calendar#142]
Before 2.0 can be released, a thorough round of back-compat testing needs to be performed. I'll include some basic components that should be checked as completed only once we are satisfied with the results.

- [x] Post types
- [x] Taxonomies
- [x] Shortcodes
- [x] Widgets
- [x] Feeds
- [x] Function names & calls

Please comment here with the results of testing, bugs, issues, and so on.

**Comment by JJJ** [comment]
Merged progress from #144. This is necessary because the post type and taxonomy changed throughout the branch, back to support data from 1.x. New branches from `release/2.0` should include these changes, to avoid the rework.

**Comment by JJJ** [comment]
I've tested everything listed here as best as possible myself.

Closing this in favor of new issues for individual bugs and things that specifically need doing.
